consensus/ethash/sealer.go

```diff
@@ -46,18 +47,53 @@ var (
 	errInvalidSealResult = errors.New("invalid or stale proof-of-work solution")
 )
 
+// makeFakeDelay uses the ethash.threads value as a mean time (lambda)
+// for a Poisson distribution, returning a random value from
+// that discrete function. I think a Poisson distribution probably
+// fairly accurately models real world block times.
+// Note that this is a hacky way to use ethash.threads since
+// lower values will yield faster blocks, but it saves having
+// to add or modify any more code than necessary.
+func (ethash *Ethash) makeFakeDelay() float64 {
```

Comment:
@ziogaschr What do you think about breaking this into two PRs:

- an add `fakeDelay` feature which creates a fake-PoW feature that emits blocks following this distribution (no CPUs and configurable, real-world modeling rate)

- add the `dev.[ethash|pow]` flag which would activate this configuration

Reply:
Sounds reasonable. Do we want to re-open #180 and check if it's mergable? Then change this one to only include the `--dev.[ethash.pow]` flag?

Reply:
Yea, SGTM